lighthouse-core/report/html/renderer/util.js

```diff
@@ -496,7 +496,9 @@ Util.i18n = null;
  */
 Util.UIStrings = {
   /** Disclaimer shown to users below the metric values (First Contentful Paint, Time to Interactive, etc) to warn them that the numbers they see will likely change slightly the next time they run Lighthouse. */
-  varianceDisclaimer: 'Values are estimated and may vary. The [performance score is calculated](https://web.dev/performance-scoring/) directly from these metrics.',
+  varianceDisclaimer: 'Values are estimated and may vary. The [performance score is calculated](https://web.dev/performance-scoring/) directly from these metrics. ',
+  /** Text link pointing to the Lighthouse scoring calculator. This link immediately follows a sentence stating the performance score is calculated from the perf metrics. */
```

Comment:
I'm not sure if the second sentence adds much for the translators. Maybe something about it being an "interactive calculator that explains Lighthouse scoring"? Maybe unnecessary. But maybe add a suggestion about keeping it short?

Reply:
thx